pkg/controllers/dynakube/extension/eec/conditions.go

```diff
@@ -1,3 +1,3 @@
 package eec
 
-const extensionControllerStatefulSetConditionType string = "ExtensionsControllerStatefulSet"
+const ExtensionControllerStatefulSetConditionType string = "ExtensionsControllerStatefulSet"
```

Comment:
Why is the const called Extension without `s` but the value is Extensions with `s`?